test/test_sparse_csr.py

```diff
@@ -3484,6 +3485,50 @@ def test_sparse_to_sparse_compressed(self, device, dtype, coalesced, layout):
             self.assertEqual(torch.tensor(sp_matrix.indices, dtype=torch.int64), plain_indices_mth(pt_matrix))
             self.assertEqual(torch.tensor(sp_matrix.data), pt_matrix.values())
 
+    @unittest.skipIf(not TEST_CUDA_CUDSS, "The test requires cudss")
+    @dtypes(*floating_types())
+    def test_linalg_solve_sparse_csr_cusolver(self, device, dtype):
+        # https://github.com/krshrimali/pytorch/blob/f5ee21dd87a7c5e67ba03bfd77ea22246cabdf0b/test/test_sparse_csr.py
+
+        if (device == 'meta') or (device == 'cpu'):
+            self.skipTest("Skipped!")
```

Comment:
Just instantiate this test on the CUDA device and skip it as well at the top if you are not testing on CUDA.

Reply:
Thanks! Updated in https://github.com/pytorch/pytorch/pull/129856/commits/4238b479911f3f542b03cf785f97a26685243821